### PR TITLE
support for ARITH_MULT_SIGN

### PR DESCRIPTION
**Pull request by tomaz1502**
Note: purification skolems were creating nodes with "DIVISION_TOTAL" kind, which wasn't supported by the printer. This PR also adds support for this kind, treating them in the same way as "DIVISION".

### CODE REVIEW

File: src/proof/lean/lean_post_processor.cpp [hunk 1]
```diff
@@ -1304,6 +1305,75 @@ bool LeanProofPostprocessCallback::update(Node res,
...
+        Node toPush;
+        int32_t pol;
+        Assert(!currArg[0].isConst()
+               || currArg[0].getConst<Rational>() == Rational(0));
+        Assert(!currArg[1].isConst()
+               || currArg[1].getConst<Rational>() == Rational(0));
+        if (currArg[0].isConst())
+        {
+          toPush = currArg[1];
+          pol = currArg.getKind() == Kind::LT      ? 1
```
Comment (HanielB) [inline src/proof/lean/lean_post_processor.cpp:1329]: clang-format

**Review by HanielB — APPROVED** [review]
looks good! Just clang format

File: src/proof/lean/lean_post_processor.cpp (hunk 1 above)
Comment (tomaz1502) [reply src/proof/lean/lean_post_processor.cpp:1329]: hmm, this was the output of clang-format for me... can you share the specific version and which specific flag do you set?

Comment (HanielB) [reply src/proof/lean/lean_post_processor.cpp:1329]: You are right. I wrongly assumed this was not clang-formatted hehe

File: src/proof/lean/lean_post_processor.cpp [hunk 2]
```diff
@@ -1304,6 +1305,75 @@ bool LeanProofPostprocessCallback::update(Node res,
...
+    {
+      std::vector<Node> variables;
+      std::vector<Node> pols;
+      size_t numArgs = args.size();
+      // last element is the conclusion
+      for (size_t i = 0; i < numArgs - 1; i++)
+      {
+        Assert(arg[i].getKind() == Kind::NOT ||
+               arg[i].getKind() == Kind::LT  ||
+               arg[i].getKind() == Kind::GT);
```
Comment (HanielB) [inline src/proof/lean/lean_post_processor.cpp]:
```suggestion
        Assert(arg[i].getKind() == Kind::NOT || arg[i].getKind() == Kind::LT
               || arg[i].getKind() == Kind::GT);
```
this clangs formats like this for me

**Review by HanielB — APPROVED** [review]

**Pull request merged**